homeassistant/components/frontend/__init__.py

```diff
@@ -342,10 +342,12 @@ def update_theme_and_fire_event():
         """Update theme_color in manifest."""
         name = hass.data[DATA_DEFAULT_THEME]
         themes = hass.data[DATA_THEMES]
-        if name != DEFAULT_THEME and PRIMARY_COLOR in themes[name]:
-            MANIFEST_JSON["theme_color"] = themes[name][PRIMARY_COLOR]
-        else:
-            MANIFEST_JSON["theme_color"] = DEFAULT_THEME_COLOR
+        MANIFEST_JSON["theme_color"] = DEFAULT_THEME_COLOR
+        if name != DEFAULT_THEME:
+            MANIFEST_JSON["theme_color"] = themes[name].get(
+                "app-header-background-color",
```

Comment:
`app-header-background-color` is not an official supported variable for themes.

Reply:
I wasn't aware of that. What's your proposal to address this issue then? Dark / black themes are definitely beneficial for oled displays because the black pixels don't consume power.

Reply:
Isn't it? https://github.com/home-assistant/home-assistant-polymer/pull/4269

Where is `theme_color` used? Mostly in the header and navigation bar of web-app-whatchacallits, right?
I think as long as it falls back to `primary-color` by default this is a good change.

Reply:
Oh I guess it exists. In that case I think it's fine. What do you think @bramkragten ?

Reply:
I guess this will work, as it is mostly the header the like Thomas says.

Reply:
Can you cherry pick this for 105 then as it's a simple fix?